applications/crossbar/priv/couchdb/schemas/system_config.callflow.json

```diff
@@ -64,14 +64,6 @@
             "description": "callflow post answer delay",
             "type": "integer"
         },
-        "privacy_name": {
```

Comment:
@icehess you will need to add a migrate clause to move existing callflow settings for privacy to the privacy system_config doc